pkg/udev/udev.go

```diff
@@ -0,0 +1,132 @@
+// +build linux,cgo
+
+package udev
```

Comment:
Did we use this code from some other package?
If yes, lets put the license details in the header.

Reply:
This code is written in a typical idiomatic go style. This is the style we should be following. There are no nested logic statements. Hence, easy to read, understand & maintain. There are good method level comments.

Reply:
Thanks, This code is our code only.